### DISCUSSION TITLE
Order details pane for a content type

### DISCUSSION THREAD

**Discussion by rodasasamoila** [pnp/pnpcore#1295] [Q&A]
I am trying to reorder the fields on this content type in the details pane but I am not seeing any property/method in order to do it. I would like to put "Document Source" above "Document Description". Deleting the fields from the content type and adding them again in the correct order is not possible since the production environment contains information on both of those fields.
![image](https://github.com/pnp/pnpcore/assets/53436717/d7e7f982-2c24-4944-a9d7-6e252e1c9c00)

**Comment by jansenbe** [comment]
@rodasasamoila : this possibly can be done using FieldLink reordering for the contenttype, we however have not implemented that at the moment. See here for the REST API that does this: https://s-kainet.github.io/sp-rest-explorer/#/entity/Collection(SP.FieldLink)/func/Reorder
